app/models/fe/question.rb

```diff
@@ -83,7 +83,7 @@ def default_label?
       true
     end
 
-    def locked?(params, answer_sheet, presenter)
+    def locked?(params, answer_sheet, presenter, current_person)
```

Comment:
I was initially confused over this extra param because it's not used here, but of course you replace this routine in PR. Maybe a comment is in order?